fvm/src/call_manager/default.rs

```diff
@@ -165,10 +165,10 @@ where
     }
 
     fn finish(mut self) -> (FinishRet, Self::Machine) {
+        // TODO: Having to check against zero here is fishy, but this is what lotus does.
```

Comment:
I'd be pretty happy to see this turned into an error and fail the message, but...TODO for now...